store.go

```diff
@@ -2251,6 +2255,7 @@ func (s *store) Shutdown(force bool) ([]string, error) {
 		s.graphLock.Touch()
 		modified = true
 	}
+	mount.Unmount(s.RunRoot())
```

Comment:
Use `s.runRoot` instead of `s.RunRoot()` here, read the result from `Unmount()`, and set `err` to its value if `err` is not already set.